.github/workflows/run-danger.yml

```diff
@@ -0,0 +1,11 @@
+name: ☢️ Danger
+
+on:
+  pull_request:
+    types: [opened, synchronize, edited, review_requested, review_request_removed, labeled, unlabeled, milestoned, demilestoned]
+
+jobs:
+  dangermattic:
+    uses: Automattic/dangermattic/.github/workflows/reusable-run-danger.yml@iangmaia/danger-on-gha
```

Comment:
Just a note to remember to update this again once your `danger-on-gha` branch and [PR#32](https://github.com/Automattic/dangermattic/pull/32) lands in dangermattic